docs/docs/about/references/keywords/DIRECT_EMITTERS.md

```diff
@@ -0,0 +1,46 @@
+# DIRECT_EMITTERS
```

Comment:
Should we remove the lines below about version 8.7 and pre 8.6, or keep them as well?

Reply:
It would be great if we "just agree" on what works, and then make sure to be consistent with that for the following addintions, changes and removals :) I think the best would be to use a "template", but I guess that is a bit of a hazzle to make it work (template; ie special css styles, as we did for old and new yaml code in migration guide)